<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhPreferNumbers</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_CadClie">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrando Cliente!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome completo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tele">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_End">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endere√ßo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_end">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CPF">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cpf">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCad">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 146, 255);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 47, 50);</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cadastro">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 188, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_CadClie</zorder>
   <zorder>label_Nome</zorder>
   <zorder>lineEdit_nome</zorder>
   <zorder>label_Tele</zorder>
   <zorder>lineEdit_tel</zorder>
   <zorder>label_End</zorder>
   <zorder>lineEdit_end</zorder>
   <zorder>label_CPF</zorder>
   <zorder>lineEdit_cpf</zorder>
   <zorder>pushButtonCad</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagem.qrc"/>
 </resources>
 <connections/>
</ui>
